tests/scenarios/revert.spec.js

```diff
@@ -0,0 +1,142 @@
+      client.waitForElementVisible(aposCommitBtnSelector);
+      client.click(aposCommitBtnSelector);
+      client.waitForElementVisible(modalExportSelector);
+      client.waitForElementVisible(exportSkipSelector);
+      client.click(exportSkipSelector);
+      client.waitForElementNotPresent(notificationSelector);
+    }
+  },
+  {
+    'article can be found under "Articles" in draft mode with title: Article Title 1': (client) => {
+      const blogBtnSelector = '[data-apos-admin-bar-item="apostrophe-blog"]';
+      const blogTitleSelector = '.apos-manage-apostrophe-blog-title a';
+
+      client.keys(client.Keys.ESCAPE);
+      client.keys(client.Keys.ESCAPE);
+      client.waitForElementVisible(blogBtnSelector);
+      client.click(blogBtnSelector);
+      client.waitForElementVisible(blogTitleSelector);
+      client.expect.element(blogTitleSelector).text.to.equal('Article Title 1');
+    }
+  },
+  {
+    'revert to initial version': (client) => {
+      const modalExportSelector = '.apos-workflow-export-modal';
+      const modalBlogSelector = '.apostrophe-blog-manager';
+      const modalEditArticleSelector = '.apos-pieces-editor';
+      const managerSelector = '.apos-manage-table';
+      const manageTableRowSelector = '.apos-manage-table tr[data-piece]';
+      const editArticleBtnSelector = `${manageTableRowSelector} a`;
+      const workflowModalBtnSelector =
+        `${modalEditArticleSelector} [data-apos-dropdown-name="workflow"]`;
+      const notificationSelector = '.apos-notification-container';
+      const workflowHistoryBtnSelector = `[data-apos-dropdown-name=workflow] [data-apos-workflow-history]`;
+      const workflowRevertBtnSelector = ('.apos-manage-table tr:last-child td [data-apos-workflow-revert]');
+
+      client.waitForElementNotPresent(modalExportSelector);
+      client.waitForElementVisible(modalBlogSelector);
+      client.waitForElementVisible(manageTableRowSelector);
+      client.click(editArticleBtnSelector);
+      client.waitForElementVisible(modalEditArticleSelector);
+      client.click(workflowModalBtnSelector);
+      client.pause(200);
+      client.waitForElementVisible(workflowHistoryBtnSelector);
+      client.click(workflowHistoryBtnSelector);
+      client.waitForElementVisible(managerSelector);
+      client.waitForElementVisible(workflowRevertBtnSelector);
+      client.click(workflowRevertBtnSelector);
+      client.waitForElementNotVisible(notificationSelector);
+    }
+  },
+  {
+    'article can be found under "Articles" in draft mode with title: New Article Title': (client) => {
+      const blogBtnSelector = '[data-apos-admin-bar-item="apostrophe-blog"]';
+      const blogTitleSelector = '.apos-manage-apostrophe-blog-title a';
+      const notificationSelector = '.apos-notification-container';
+
+      client.keys(client.Keys.ESCAPE);
+      client.keys(client.Keys.ESCAPE);
+      client.pause(500);
```

Comment:
Do not use pause (if it won't work without pause talk to me about what I might need to fix).

Reply:
This is because of the alert - I did some research but I'm not sure how to detect the presence of the alert. I do know how to get rid of it once it's present, so we wait :)

Reply:
Fair.